DOCSP-48707 Standardize cursor chaining methods (#1073)
mongodb/docs-node: source/crud/query/specify-documents-to-return.txt

- ``limit()``: Specifies the maximum number of documents to return from a query.
- ``skip()``: Specifies the number of documents to skip before returning query results.

Cursor Chaining API versus the Options API
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

You can specify which documents to return by chaining the above methods to your read 
operation or by specifying them in an ``options`` object in your call to your 
read operation. The following example shows how to use both the chaining API and the
options API to achieve the same result:

.. code-block:: javascript

   // Uses the cursor chaining API
   myColl.find(query).sort({ length: -1 }).limit(3);

   // Uses the object API
   myColl.find(query, { sort: { length: -1 }, limit: 3 });

> We recommend that you use the cursor chaining API for [what is the reason]. The 

Author: Will not mention options api instead.

     { "_id": 6, "name": "A Dance With Dragons", "author": "Martin", "length": 1104 },
   ]);

.. note::

>    When you chain ``sort()``, ``limit()``, or ``skip()`` to a read operation, you must
>    specify all methods before iterating the cursor. If you specify a method after
>    iterating the cursor, the method you specified does not apply to the operation.

Reviewer: S; the order of ideas in this note is hard to follow. Maybe something like this is clearer:
```suggestion
   You must chain methods such as ``sort()``, ``limit()``, or ``skip()`` to a read operation
   before iterating the cursor. If you specify an option method after
   iterating the cursor, the setting does not apply to the operation.
```

``"1104"``. The results are sorted in alphabetical order, skipping the first
document and includes only the first result:

.. io-code-block::

   .. input:: 
      :language: javascript

      const query = {length: "1104"};

      const cursor = myColl.find(query).sort({ title: 1 }).skip(1).limit(1);

      for await (const doc of cursor) {
         console.dir(doc);
      }

   .. output::
      :language: json
      :visible: false
      
      { "_id": 4, "title": "Infinite Jest", "author": "Wallace", "length": 1104 }

.. note::

   The order in which you call these methods doesn't change the documents

>    that are returned. The driver automatically reorders the calls to perform the
>    sort and skip operations first, and the limit operation afterward.

Reviewer 2: ```suggestion
   that are returned.
```

This isn't accurate - each chaining method just corresponds to an option that we send to the server.  ex: `.limit()` sets the `limit` option
